bundles/core/org.eclipse.smarthome.core.semantics/src/main/resources/tags_nl.properties

```diff
@@ -0,0 +1,79 @@
+Location_Indoor=Binnen
+Location_Indoor_Building=Gebouw,Huis,Woning
+Location_Indoor_Building_Garage=Garage
+Location_Indoor_Floor=Verdieping,Etage,Vloer
+Location_Indoor_Floor_GroundFloor=Begane Grond,Beneden,Beneden Etage
+Location_Indoor_Floor_FirstFloor=Eerste Verdieping,Eerste Etage,Boven
+Location_Indoor_Floor_Attic=Zolder
+Location_Indoor_Floor_Basement=Kelder
+Location_Indoor_Corridor=Hal,Gang
+Location_Indoor_Room=Kamer,Vertrek,Ruimte,Verblijf
+Location_Indoor_Room_Bedroom=Slaapkamer,Slaapkamers
+Location_Indoor_Room_Kitchen=Keuken
+Location_Indoor_Room_Bathroom=Badkamer
+Location_Indoor_Room_LivingRoom=Woonkamer
+Location_Outdoor=Buiten
+Location_Outdoor_Garden=Tuin
+Location_Outdoor_Terrace=Terras,Patio
+Location_Outdoor_Carport=Carport
+Property_Temperature=Temperatuur,Temperaturen
+Property_Light=Licht,Lichten
+Property_ColorTemperature=Kleur Temperatuur
+Property_Humidity=Vochtigheid
+Property_Presence=Aanwezigheid
+Property_Pressure=Druk
+Property_Smoke=Rook
+Property_Noise=Lawaai
+Property_Rain=Regen
+Property_Wind=Wind
+Property_Water=Water
+Property_CO2=CO2,Koolstof Dioxide
+Property_CO=CO,Koolstof Monoxide
+Property_Energy=Energie
+Property_Power=Vermogen
+Property_Voltage=Voltage
+Property_Current=Stroom
+Property_Frequency=Frequentie
+Property_Gas=Gas
+Property_SoundVolume=Geluidsvolume
+Property_Oil=Olie
+Point_Alarm=Alarm
+Point_Control=Regelaar
+Point_Control_Switch=Schakelaar
+Point_Measurement=Meeting
+Point_Setpoint=Instelpunt
+Point_Status=Status
+Point_Status_LowBattery=Bijna Lege Batterij
+Point_Status_OpenState=Open,Open Toestand
+Point_Status_Tampered=Geknoeid
+Point_Status_OpenLevel=Open niveau
+Point_Status_Tilt=Kantelen
+Equipment_Battery=Batterij,Batterijen
+Equipment_Blinds=Zonnewering,Jaloezien,Luxaflex,Lamellen,Rolgordijn
+Equipment_Camera=Camera,Camera's
+Equipment_Car=Auto,Auto's
```

Comment:
Will the single apostrophe work? GitHub now also adds a blue color. 
Perhaps it should be escaped by adding an apostrophe (i.e. double apostrophes)?

Reply:
Agree properties files should have them escaped